modules/kubernetes/pages/k8s-operator/backup-and-restore.adoc

```diff
@@ -0,0 +1,710 @@
+If destination is s3:
+--s3-bucket :               S3 Bucket name
+--aws-access-key-id :       the aws access key id
+--aws-secret-access-key :   the aws secret access key
+[source.wrap, console]
+----
+
+You can restore your cluster from the backup created by the same cluster.
+This works with backups stored in either local storage or an S3 bucket.
+
+We also support cross-cluster restore which means, restore Cluster B from backup created Cluster A. This only supports S3 bucket now.
+
+Notes: now we just supports restore cluster having the same size and ha as the cluster which created the backup. If you create backup in Cluster A whose size is 4, you cannot restore Cluster B whose size is 8 from the backup created by Cluster A.
+
+=== Restore in the same cluster
+
+Assume that you have created a backup for test-cluster using kubectl tg backup create.
+Use the following command to get the tags for all backups:
+
+[source.wrap, console]
+----
+kubectl tg backup list --cluster-name test-cluster -n tigergraph
+
+
++------------------------------+------+---------+--------+---------------------+
+|             TAG              | TYPE | VERSION |  SIZE  |     CREATED AT      |
++------------------------------+------+---------+--------+---------------------+
+| daily-2022-11-02T103601      | FULL | 3.9.0   | 1.7 MB | 2022-11-02 10:36:02 |
+| daily-2022-11-02T104925      | FULL | 3.9.0   | 1.7 MB | 2022-11-02 10:49:25 |
+| daily-2022-11-09T081545      | FULL | 3.9.0   | 1.7 MB | 2022-11-09 08:15:46 |
+| daily-2022-11-09T081546      | FULL | 3.9.0   | 1.7 MB | 2022-11-09 08:15:53 |
++------------------------------+------+---------+--------+---------------------+
+----
+
+Choose a backup you want to restore from and provide the storage information of the backup.
+If you want to use a backup stored in local storage, provide the flags `--source local` and `--local-path`.
+
+If you want to use a backup stored in S3, provide the flags `--source s3Bucket` and `--s3-bucket --aws-access-key-id  --aws-secret-access-key`.
+
+=== Restore a backup from local storage
+
+[source.wrap, console]
+----
+kubectl tg restore --cluster-name test-cluster -n tigergraph --tag daily-2022-11-02T103601\
+--source local --local-path /home/tigergraph/backup
+----
+
+=== Restore a backup from an S3 bucket
+
+[source.wrap, console]
+----
+kubectl tg restore --namespace tigergraph --cluster-name test-cluster \
+--tag tests3-2022-10-31T031005 \
+--source s3Bucket --s3-bucket tg-backup --aws-access-key-id AWS_ACCESS_KEY \
+--aws-secret-access-key AWS_SECRET
+----
+
+=== Cross-cluster restore
+
+Cross-cluster restoration is when you restore a existing cluster from a backup created by another cluster.
+
+Assume that you have created a backup for a cluster called `source-cluster`, and the backup is stored in an S3 bucket.
+Your target cluster, here named `target-cluster` must have the same size, ha and version as `source-cluster`.
```

Comment:
```suggestion
Your target cluster, here named `target-cluster` must have the same size, HA replication factor, and version as `source-cluster`.
```

Reply:
"ha" is not a measurable property such that two clusters can "have the same HA" (improper use of "HA").

Reply:
"Version": Version of what? TigerGraph database software?